notes/notes.proto

```diff
@@ -88,22 +88,31 @@ message ListNotesReply {
 }
 
 message AddBlockRequest {
+    //the note id where we add a block
+    string note_id = 1;
+
     // The block to append to the note.
-    Block block = 1;
+    Block block = 2;
 
     // Given that a note is an array of blocks, provide the index at
     // which the block shall be inserted.
-    uint32 index = 2;
+    uint32 index = 3;
 }
 
 message UpdateBlockRequest {
-    string id = 1;
+    // The note id where we add a block
+    string note_id = 1;
```

Comment:
I don't get it, how do you know which block to update if you only pass in a `note_id`. I think we should keep the `id` as it was.